docs/GettingStarted.md

```diff
@@ -1,104 +1,160 @@
 # Getting Started With `redux-form`
 
-`redux-form` primarily consists of four things: 
+The basic implementation of `redux-form` is simple. However, to make the most of it, it's recommended to have basic knowledge on:
+* [Redux](http://redux.js.org/) state container,
+* [React](https://facebook.github.io/react/) and [Higher Order Components (HOCs)](https://facebook.github.io/react/docs/higher-order-components.html).
 
-1. A Redux reducer that listens to dispatched `redux-form` actions to maintain your form state in
-Redux.
-2. A React component decorator that wraps your entire form in a Higher Order Component (HOC) and 
-provides functionality via props.
-3. A `Field` component to connect your individual field inputs to the Redux store.
-4. Various Action Creators for interacting with your forms throughout the application.
+## Overview
 
-## Implementation Guide
+To connect your React form components to your Redux store you'll need following pieces:
 
-### Step #1
+* Redux Reducer: `formReducer`,
+* React HOCs: `reduxForm()` and `<Field/>`.
```

Comment:
Hmm.... I never considered `Field` to be a higher order component, but I suppose it sort of is....